app/learning.js

```diff
@@ -7,51 +7,86 @@
  * Main script for initiating the learning app.
  */
 
-define(['jquery', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer','app/database'], function ($, bootstrap, config, db, messages, question, timer,db) {
-  timer.startTimer(".timer", config.constant("TIME_LIMIT"));
+/*jshint esversion: 6 */
+define(['jquery', 'app/lang', 'app/string', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer', 'app/ready', 'app/user', 'app/time'], function ($, lang, string, bootstrap, config, db, messages, questions, timer, ready, user, time) {
+
+  const THOUSAND = 1000;
   var waitingForEnter = false;
-  
+
   function disableAutocomplete() {
     $('input').attr('autocomplete', 'off');
   }
-  
+
+	function inputIsEmpty() {
+		return $.trim($("#answer").val()).length === 0;
+	}
+
   function nextQuestion() {
     clearTimeout(timeout);
     timer.clearCountdown();
     messages.clear();
+    questions.nextQuestion();
     $( "#answer" ).prop("disabled", false);
```

Comment:
What do all these changes on the element with id answer do? Use a function to tell me.